utils/controleval.py

```diff
@@ -82,19 +89,34 @@ def calculate_stats(ctrls):
     print_specific_stat("Applicable", applicable, total)
     print_specific_stat("Assessed", assessed, applicable)
     print()
-    print_specific_stat("Automated", ctrlstats[controls.Status.AUTOMATED], applicable)
-    print_specific_stat("Manual", ctrlstats[controls.Status.MANUAL], applicable)
-    print_specific_stat("Supported", ctrlstats[controls.Status.SUPPORTED], applicable)
-    print_specific_stat("Documentation", ctrlstats[controls.Status.DOCUMENTATION], applicable)
-    print_specific_stat("Inherently Met", ctrlstats[controls.Status.INHERENTLY_MET], applicable)
-    print_specific_stat("Does Not Meet", ctrlstats[controls.Status.DOES_NOT_MEET], applicable)
-    print_specific_stat("Partial", ctrlstats[controls.Status.PARTIAL], applicable)
+    print_specific_stat("Automated",
+                        ctrlstats[controls.Status.AUTOMATED],
+                        applicable)
+    print_specific_stat("Manual",
+                        ctrlstats[controls.Status.MANUAL],
+                        applicable)
+    print_specific_stat("Supported",
+                        ctrlstats[controls.Status.SUPPORTED],
+                        applicable)
+    print_specific_stat("Documentation",
+                        ctrlstats[controls.Status.DOCUMENTATION],
+                        applicable)
+    print_specific_stat("Inherently Met",
+                        ctrlstats[controls.Status.INHERENTLY_MET],
+                        applicable)
+    print_specific_stat("Does Not Meet",
+                        ctrlstats[controls.Status.DOES_NOT_MEET],
+                        applicable)
+    print_specific_stat("Partial",
+                        ctrlstats[controls.Status.PARTIAL],
+                        applicable)
 
     applicablelist = ctrls - ctrllist[controls.Status.NOT_APPLICABLE]
     assessedlist = set().union(ctrllist[controls.Status.AUTOMATED]).union(ctrllist[controls.Status.SUPPORTED])\
         .union(ctrllist[controls.Status.DOCUMENTATION]).union(ctrllist[controls.Status.INHERENTLY_MET])\
         .union(ctrllist[controls.Status.PARTIAL]).union(ctrllist[controls.Status.MANUAL])
-    print("Missing:", ", ".join(sorted(str(c.id) for c in applicablelist - assessedlist)))
+    print("Missing:", ", ".join(sorted(str(c.id)
+          for c in applicablelist - assessedlist)))
 
 
 def print_specific_stat(stat, current, total):
```

Comment:
Should this be included in the JSON formatted output?

Reply:
Do we want to add the percentages to the JSON representation? Or is the thing consuming the JSON going to do that if needed?

Reply:
The percentage will be calculated at the other end